packages/title-case/src/index.ts

```diff
@@ -1,7 +1,7 @@
 const TOKENS = /(\S+)|(.)/g;
 const IS_SPECIAL_CASE = /[\.#]\p{Alphabetic}/u; // #tag, example.com, etc.
 const IS_MANUAL_CASE = /\p{Ll}(?=[\p{Lu}])/u; // iPhone, iOS, etc.
-const ALPHANUMERIC_PATTERN = /\p{Alphabetic}+/gu;
+const ALPHANUMERIC_PATTERN = /[\p{Alphabetic}\p{Nd}]+/gu;
```

Comment:
Any concerns with just doing `\p{N}`?

Reply:
No concerns with changing to `\p{N}`

Reply:
Release yesterday, cheers! https://github.com/blakeembrey/change-case/releases/tag/title-case%404.3.2